addons/l10n_ch/tests/test_l10n_ch_isr.py

```diff
@@ -83,11 +85,13 @@ def test_isr(self):
         test_account = self.create_account('250097798')
         invoice_1.partner_bank_id = test_account
         self.isr_not_generated(invoice_1)
+        invoice_1.partner_bank_id.write({
+            'l10n_ch_isr_subscription_chf': 'CHF'
```

Comment:
This is a wrong value for that field.

This field must contain a ISR subscription number such as `01-162-8`

Reply:
@yvaucher  This test case has been fixed in this commit https://github.com/odoo/odoo/commit/161796e2d9895dfe1508ff233596d2ae95fa5c9f
Thanks